kong/runloop/plugins_iterator.lua

```diff
@@ -82,7 +81,7 @@ local PluginsIterator = {}
 -- @tparam string|nil consumer_id The consumer identifier. If `nil`, an empty string is used.
 -- @treturn string The compound key, in the format `route_id:service_id:consumer_id`.
 local function build_compound_key(route_id, service_id, consumer_id)
```

Comment:
Just trying to understand the code and new to Lua.
Is there any reason to keep this function if we are not using it and the scope is local as well ?

Reply:
@SauravPrakash98 it is used during building new plugin iterator, just not used on request time.